curator/utils.py

```diff
@@ -1189,8 +1184,26 @@ def snapshot_running(client):
     :arg client: An :class:`elasticsearch.Elasticsearch` client object
     :rtype: bool
     """
+    status = []
+
     try:
-        status = client.snapshot.status()['snapshots']
+        # Check to see if we're running AWS Elasticsearch and if so, iterate through each repository to check for running snapshots
```

Comment:
Is this still necessary? [`/_snapshot/_status` endpoint is working after AWS ES add support for 5.5](https://github.com/elastic/curator/issues/987#issuecomment-333255862)

Reply:
I have tested the original code agains an AWS ES 5.3 and 5.5 cluster and it works as expected now, so this is not necessary anymore.